src/components/pages/home/tickets/tickets.jsx

```diff
@@ -1,22 +1,109 @@
+            <SpeakerProfileImage speakers={workshopSpeakers} speakerId={"cb938518-75cf-4ead-9262-f24323657e4c"} className="max-h-16 rounded-b-full cursor-pointer" onClick={() => handleImageClick("cb938518-75cf-4ead-9262-f24323657e4c")} />
+          </div>
+          <div className="pl-10 flex flex-col gap-1 items-start text-start" >
+            <strong className="text-sm text-start bg-pink text-white py-1 px-2 rounded-full inline-block self-start">Room 2, 09:00-12:00</strong>
+            <h4 className="text-purple font-bold text-balance cursor-pointer" onClick={() => handleSessionClick("701299")}>Detecting unexpected behavior and intrusions with Falco + Atomic Red Team</h4>
+            <strong>Miguel Hernández</strong>
+            <span className="text-balance text-gray-5 text-sm">Staff Threat Researcher Engineer @ Sysdig</span>
+            <strong>Vicente J. Jiménez Miras</strong>
+            <span className="text-balance text-gray-5 text-sm">Developer Advocate &amp; Technical Trainer</span>
+          </div>
+        </div>
+        <div className="flex items-center m-4 ml-4 p-2 relative bg-gray-12 rounded-md">
+          <SpeakerProfileImage speakers={workshopSpeakers} speakerId={"1829d862-9dc7-49f7-b290-0f69bf61f7d0"} className="absolute -left-8 max-h-16 rounded-full cursor-pointer" onClick={() => handleImageClick("1829d862-9dc7-49f7-b290-0f69bf61f7d0")} />
+          <div className="pl-10 flex flex-col gap-1 items-start text-start">
+            <strong className="text-sm text-start bg-pink text-white py-1 px-2 rounded-full inline-block self-start">Room 1, 13:00-16:00</strong>
+            <h4 className="text-purple font-bold text-balance cursor-pointer" onClick={() => handleSessionClick("712278")}>Deep Dive Into Kubernetes IAM and Network Security: How To Automate at Scale With IBAC and GitOps</h4>
+            <strong>Nic Vermande</strong>
+            <span className="text-balance text-gray-5 text-sm">Head of DevRel @ Otterize</span>
+          </div>
+        </div>
+        <div className="flex items-center m-4 ml-4 p-2 relative bg-gray-12 rounded-md">
+          <SpeakerProfileImage speakers={workshopSpeakers} speakerId={"a2665c2b-13c9-4337-9c78-db85bca70e60"} className="absolute -left-8 max-h-16 rounded-full cursor-pointer" onClick={() => handleImageClick("a2665c2b-13c9-4337-9c78-db85bca70e60")} />
+          <div className="pl-10 flex flex-col gap-1 items-start text-start">
+            <strong className="text-sm text-start bg-pink text-white py-1 px-2 rounded-full inline-block self-start">Room 2, 13:00-16:00</strong>
+            <h4 className="text-purple font-bold text-balance cursor-pointer" onClick={() => handleSessionClick("697243")}>Kubernetes Performance Tuning</h4>
+            <strong>Ant(on) Weiss</strong>
+            <span className="text-balance text-gray-5 text-sm">Software Delivery Futurist</span>
+          </div>
```

Comment:
```suggestion
          <div className="pl-10 flex flex-col content-between gap-1 h-full">
            <strong className="text-sm text-start bg-pink text-white py-1 px-2 rounded-full inline-block self-start">Room 2, 13:00-16:00</strong>
            <div className="flex flex-col gap-1 items-start text-start justify-center h-full">
              <h4 className="text-purple font-bold text-balance">Kubernetes Performance Tuning</h4>
              <strong>Ant(on) Weiss</strong>
              <span className="text-balance text-gray-5 text-sm">Software Delivery Futurist</span>
            </div>
          </div>
```
This would align the pill left with the text but on top of the card
![image](https://github.com/user-attachments/assets/9f77242c-db8b-4268-bc6f-9c59e9ece43d)